netlify.toml

```diff
@@ -78,9 +78,6 @@
 [[redirects]]
   from = "/crowdsale"
   to = "/en/developers/"
-[[redirects]]
-  from = "/dao"
-  to = "/en/developers/"
```

Comment:
Would it make sense to redirect to https://ethereum.org/en/dao, or is the point of removing this redirect to account for translated pages in other languages?

Reply:
Agreed, I'm in favor of having it redirect to the english pages for the use cases, ie:

```
https://ethereum.org/dao => https://ethereum.org/en/dao
https://ethereum.org/nft => https://ethereum.org/en/nft
https://ethereum.org/defi => https://ethereum.org/en/defi
```

Perhaps I'll open an issue to just re-visit this entire `netlify.toml` file and clean up all the redirects, especially as we're planning future content versions. In the meantime will bring this patch in.

Reply:
Actually, now that I look at this, we're removing a redirect and not replacing it with anything... I'll just add the use-cases redirects I just mentioned to this PR.